### PR TITLE
[PLAT-287] Add lookAt shortcut

### PR DESCRIPTION
**Pull request by jdm717**
## Summary

Adds support for a "Fly to Position" keybind (`Alt` + `Shift`), and modifies the existing "Fly to Part" keybind (`Alt` + !`Shift`).

https://user-images.githubusercontent.com/5252480/147977038-6bc78afa-646a-4cb0-8958-8550157f2ef7.mov

## Test Plan

- Test fly to part (`Alt` + Click)
- Test fly to position (`Alt` + `Shift` + Click)

## Release Notes

N/A

## Possible Regressions

Fly to part

## Dependencies

N/A

### CODE REVIEW

File: packages/viewer/src/lib/interactions/flyToPartKeyInteraction.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ export class FlyToPartKeyInteraction
   ) {}
 
   public predicate(e: TapEventDetails): boolean {
-    return !!e.altKey;
+    return !!e.altKey && !e.shiftKey;
```
Comment (amvertex) [inline packages/viewer/src/lib/interactions/flyToPartKeyInteraction.ts:18]: Do you want two exclamation marks for the shiftKey part as well?

**Review by amvertex — APPROVED** [review]

File: packages/viewer/src/lib/interactions/flyToPositionKeyInteraction.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+{
+  public constructor(
+    private stream: StreamApi,
+    private configProvider: ConfigProvider,
+    private imageScaleProvider: ImageScaleProvider,
+    private sceneProvider: SceneProvider
+  ) {}
+
+  public predicate(e: TapEventDetails): boolean {
+    return !!e.altKey && !!e.shiftKey;
```
Comment (danschultz) [inline packages/viewer/src/lib/interactions/flyToPositionKeyInteraction.ts:21]: `e.altKey && e.shiftKey`?

File: packages/viewer/src/lib/interactions/flyToPartKeyInteraction.ts (hunk 1 above)
Comment (danschultz) [inline packages/viewer/src/lib/interactions/flyToPartKeyInteraction.ts:18]: `e.altKey && !e.shiftKey`?

Comment (jdm717) [reply packages/viewer/src/lib/interactions/flyToPartKeyInteraction.ts]: Updated in https://github.com/Vertexvis/vertex-web-sdk/pull/313/commits/e3727dc4690dd42f62f6247a6e3ba39713e205b6. Copied over the `!!` syntax from the existing fly to part interaction, but it does look like that's unnecessary now

**Review by therockstorm — APPROVED** [review]

**Review by danschultz — APPROVED** [review]

**Pull request merged**